x-pack/plugins/exploratory_view/public/components/shared/filter_value_label/filter_value_label.tsx

```diff
@@ -11,6 +11,7 @@ import { Filter, buildPhrasesFilter, buildPhraseFilter } from '@kbn/es-query';
 import { FilterItem } from '@kbn/unified-search-plugin/public';
 import type { DataView } from '@kbn/data-views-plugin/common';
 import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { ObservabilityAppServices } from '../../../application/types';
```

Comment:
Could you please use ObservabilityPluginStart instead? This type is a duplicate and we want to remove it with https://github.com/elastic/kibana/pull/158144.

Reply:
@CoenWarmer Looks like there is no `ObservabilityPluginStart` in `x-pack/plugins/exploratory_view`. Would it be okay to keep `ObservabilityAppServices` for this PR?

Reply:
@jughosta I mean ObservabilityPublicPluginsStart: x-pack/plugins/observability/public/plugin.ts (line 90). 

I'm approving now as I am on paternity leave and don't want to block. But I would prefer if you could apply the change requested.

Reply:
Hi @elastic/actionable-observability,

Could you please help me understand your code setup so I can improve the PR changes.

From my observation there are 2 `ObservabilityAppServices` types: in `observability` and  in `exploratory_view` plugins. They include different services btw. And there is also `ObservabilityPublicPluginsStart` which might replace `ObservabilityAppServices` in `observability` via https://github.com/elastic/kibana/pull/158144 

My change is in the second `exploratory_view` plugin. I am not familiar with how your code is organized in plugins but from the high level it does not seem right to use Kibana services type from one plugin inside another plugin (as suggested `ObservabilityPublicPluginsStart` from `observability` in `exploratory_view`). Also `ObservabilityPublicPluginsStart` does not include `docLinks` as it comes from `core`. 

How would you suggest to proceed with adding `docLinks` for `useKibana` call in `exploratory_view` plugin?